cli/src/ui.rs

```diff
@@ -46,6 +48,12 @@ use crate::formatter::PlainTextFormatter;
 
 const BUILTIN_PAGER_NAME: &str = ":builtin";
 
+pub const STYLES: Styles = Styles::styled()
+    .header(AnsiColor::Yellow.on_default().bold())
+    .usage(AnsiColor::Yellow.on_default().bold())
+    .literal(AnsiColor::Green.on_default().bold())
+    .placeholder(AnsiColor::Green.on_default());
```

Comment:
nit: I would move this to `commands/mod.rs` or `cli_util.rs`. `Ui` has no dependency on `clap`.